osf/models/registration_bulk_upload.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from django.db import models
+
+from osf.models.base import BaseModel
+from osf.utils.fields import NonNaiveDateTimeField
+
+logger = logging.getLogger(__name__)
+
+
+class RegistrationBulkUpload(BaseModel):
+    """Defines a database table of which each row stores information about one registration bulk upload.
+    """
+
+    # The hash of the CSV template payload
+    payload_hash = models.CharField(blank=False, null=False, unique=True, max_length=255)
+
+    # The status/state of the bulk upload
+    state = models.CharField(choices=[
+        ('pending', 'Database preparation in progress'),
+        ('initialized', 'Database preparation done'),
+        ('picked_up', 'Registration creation in progress'),
+        ('done_full', 'Successful'),
+        ('done_partial', 'Partial'),
+        ('done_error', 'Failed'),
+    ], max_length=255)
+
+    # The user / admin who started this bulk upload
+    initiator = models.ForeignKey('OSFUser', null=True, on_delete=models.CASCADE)
```

Comment:
`blank=False` by default, but since you include it in line 35, might as well add it here.

Reply:
Good catch! I must have missed it here and there. I prefer to have these default values explicitly passed as well.